Apps/W1/HybridGP/app/src/tables/GPCompanyAdditionalSettings.table.al

```diff
@@ -320,6 +320,77 @@ table 40105 "GP Company Additional Settings"
             DataClassification = SystemMetadata;
             InitValue = true;
         }
+        field(35; "Skip All Posting"; Boolean)
+        {
+            InitValue = false;
+            DataClassification = SystemMetadata;
+
+            trigger OnValidate()
+            begin
+                if Rec."Skip All Posting" then begin
```

Comment:
Ah. The table sets the other Skip fields. If this is reflected in the UI then we are better. Can the user still change one of the other Skip values in the UI after this?

Reply:
Yes, the table handles this logic and the UI reacts accordingly.
When one of the other states change to false, the Skip All will then become false.